master/buildbot/www/auth.py

```diff
@@ -113,17 +113,19 @@ def __init__(self, header=None, headerRegex=None, **kwargs):
         if header is not None:
             self.header = header
```

Comment:
It might be worth change this line to:
       `self.header = unicode2bytes(header)`

Reply:
Damn, you're right, I don't know why I went this way, this is all a mess. It's way simplier to convert the header from string to bytes from the start and change nothing else. I'll start over!